packages/office-ui-fabric-react/src/components/pickers/Suggestions/SuggestionsController.ts

```diff
@@ -81,6 +87,22 @@ export class SuggestionsController<T> {
     this.suggestions.splice(index, 1);
   }
 
+  public createGenericSuggestion(name: string, currentValidationState: validationState) {
```

Comment:
SuggestionController isn't just for peoplepickers, it's for all pickers. You shouldn't call getInitials here. CreateGenericSuggestion should probably be another pass through renderer like RenderItem